R/checks.R

```diff
@@ -1,6 +1,6 @@
 #' Validate data input
 #'
-#' @description
+#' @description `r lifecycle::badge("stable")`
```

Comment:
Do we need these for the internal functions? I think this function will change with #346.

Reply:
Hmm no probably no need for internal as they aren't added to the pkdown index anyway.

Reply:
I'll leave these in as 1) internal/exported status might change and even if not 2) they might be useful for ourselves.

Reply:
Agreed.